docs/index.md

```diff
@@ -1,12 +1,32 @@
 Robot Interfaces Documentation {#mainpage}
 ==========================================
 
-This documentation is still work in progress.  More content will be added soon.
+
+This is the documentation of the `robot_interfaces` package.
+
+The source code is hosted on
+[GitHub](https://github.com/open-dynamic-robot-initiative/robot_interfaces).
+Please also use the issue system there if you have a question or want to report
+a bug.
+
+For more information, on the general architecture of the software, see also our
+[paper](https://arxiv.org/abs/2008.03596) on the open-source version of the
+TriFinger robot.
 
 
 Content
 -------
 
-- [Build Instructions](@ref md_docs_installation)
-- [Quick Start Example](@ref md_docs_quick_start_example)
-- [About Time Series](@ref md_docs_timeseries)
+- @ref md_docs_installation
+- @ref md_docs_quick_start_example
+- @ref md_docs_timeseries
```

Comment:
i would put the timeseries before the quick start example, since this example cannot be understood just like that. i would also change the title of this, since it does not only describe the timeseries.

i would maybe call it Logic of Actions and Observations